deeplearning4j-keras/src/test/java/org/deeplearning4j/keras/DeepLearning4jEntryPointTest.java

```diff
@@ -1,97 +1,131 @@
-        // Given
-        final File model = prepareResource("theano_mnist/model.h5");
-        final Path features = prepareFeatures("theano_mnist");
-        final Path labels = prepareLabels("theano_mnist");
-
-        EntryPointFitParameters entryPointParameters = EntryPointFitParameters
-                .builder()
-                .modelFilePath(model.getAbsolutePath())
-                .trainFeaturesDirectory(features.toAbsolutePath().toString())
-                .trainLabelsDirectory(labels.toAbsolutePath().toString())
-                .batchSize(128)
-                .nbEpoch(2)
-                .type(KerasModelType.SEQUENTIAL)
-                .build();
-
-        // When
-        deepLearning4jEntryPoint.fit(entryPointParameters);
-
-        // Then
-        // fall through - the rule will fail the test execution if an exception is thrown
-    }
-
-    private Path prepareLabels(String dataSet) throws IOException {
-        return prepareDataSet(dataSet, "labels");
-    }
-
-    private Path prepareFeatures(String dataSet) throws IOException {
-        return prepareDataSet(dataSet, "features");
-    }
-
-    private Path prepareDataSet(String dataSet, String batchesSubset) throws IOException {
-        Set<String> batchFiles = listBatchFiles(dataSet + "/" + batchesSubset);
-        final Path tempDirectory = Files.createTempDirectory("dl4j-" + batchesSubset);
-
-        for (String batchFile : batchFiles) {
-            String batchFileName = Paths.get(batchFile).getFileName().toString();
-            copyClasspathResourceToFile(batchFile, targetFile(tempDirectory, batchFileName));
-        }
-        return tempDirectory;
-    }
-
-    private File targetFile(Path tempFeaturesDirectory, String batchFileName) {
-        return Paths.get(
-                tempFeaturesDirectory.toFile().getAbsolutePath().toString(),
-                batchFileName
-        ).toFile();
-    }
-
-    private Set<String> listBatchFiles(String classpathDirectory) {
-        return new Reflections(classpathDirectory, new ResourcesScanner()).getResources(endsWith(".h5"));
-    }
-
-    private File prepareResource(String resourceName) throws IOException {
-        File file = File.createTempFile("dl4j", "test");
-        file.delete();
-        file.deleteOnExit();
-        copyClasspathResourceToFile(resourceName, file);
-        return file;
-    }
-
-    private void copyClasspathResourceToFile(String resourceName, File file) throws IOException {
-        FileUtils.copyInputStreamToFile(this.getClass().getClassLoader().getResourceAsStream(resourceName), file);
-    }
-
-}
\ No newline at end of file
+//package org.deeplearning4j.keras;
+//
+//import org.apache.commons.io.FileUtils;
+//import org.deeplearning4j.keras.api.FitParams;
+//import org.deeplearning4j.keras.api.SaveParams;
+//import org.deeplearning4j.keras.model.KerasModelType;
+//import org.junit.Rule;
+//import org.junit.Test;
+//import org.junit.rules.ExpectedException;
```

Comment:
No, I will remove it. It is a better idea to execute the tests from the Python side anyways, since that will guarantee the environment will work.